tests/sanity/tests/model/recruiting/applications-page.ts

```diff
@@ -90,7 +90,27 @@ export class ApplicationsPage extends CommonPage {
     ).toHaveText(done)
   }
 
+  async checkApplicationState (talentName: TalentName, done: string): Promise<void> {
+    await expect(
+      await this.page
+        .locator('span.ap-label', { hasText: `${talentName.lastName} ${talentName.firstName}` })
+        .locator('xpath=../../../../..')
```

Comment:
ouch, could we omit such xpath?

Reply:
Will try now =)
The main point here is to find the table row by talent name and access to another td's.

Reply:
updated!
Thank you!